AABB_tree/include/CGAL/AABB_traits_construct_by_sorting.h

```diff
@@ -0,0 +1,83 @@
+#ifndef AABB_TREE_TESTS_AABB_TRAITS_CONSTRUCT_BY_SORTING_H
+#define AABB_TREE_TESTS_AABB_TRAITS_CONSTRUCT_BY_SORTING_H
+
+#include <CGAL/AABB_traits.h>
+#include <CGAL/hilbert_sort.h>
+#include <CGAL/spatial_sort.h>
```

Comment:
Do you need spatial_sort?

Reply:
Maybe for the adapter.

Reply:
Good catch, I have a couple of includes that aren't necessary anymore.